Author: Joachim Reichel <reichel@debian.org>
Description: Improve --help output and manpage to make clear that
  normalize-audio itself only deals with WAV and MP3 files (see: #734843).
Index: normalize-audio-0.7.7/doc/normalize.1
===================================================================
--- normalize-audio-0.7.7.orig/doc/normalize.1
+++ normalize-audio-0.7.7/doc/normalize.1
@@ -6,20 +6,20 @@
 .TH "NORMALIZE\-AUDIO" "1" "14 September 2005" "" ""
 
 .SH NAME
-normalize\-audio \- adjusts volume levels of audio files.
+normalize\-audio \- adjusts volume levels of WAV or MP3 audio files.
 .SH SYNOPSIS
 
 \fBnormalize\-audio\fR [ \fB\fIoptions\fB\fR ] [ \fB\-\-\fR ] \fB\fIfile\fB\fR\fI ...\fR
 
 .SH "DESCRIPTION"
 .PP
-\fBnormalize\-audio\fR is used to adjust the volume of wav audio
+\fBnormalize\-audio\fR is used to adjust the volume of WAV or MP3 audio
 files to a standard volume level.  This is useful for things like
 creating mp3 mixes, where different recording levels on different
 albums can cause the volume to vary greatly from song to song.
 .PP
 \fBnormalize\-audio\fR operates in two phases.  In the first phase, it
-analyzes the specified files as wav audio files, and computes the
+analyzes the specified files as WAV audio files, and computes the
 volume of each file.  In the second phase, it applies a volume
 adjustment to each file to set each file's volume to a standard level.
 .SH "OPTIONS"
Index: normalize-audio-0.7.7/src/normalize.c
===================================================================
--- normalize-audio-0.7.7.orig/src/normalize.c
+++ normalize-audio-0.7.7/src/normalize.c
@@ -106,7 +106,7 @@ usage()
 {
   printf(_("\
 Usage: %s [OPTION]... [FILE]...\n\
-Normalize volume of multiple audio files\n\
+Normalize volume of multiple WAV or MP3 audio files\n\
 \n\
   -a, --amplitude=AMP          normalize the volume to the target amplitude\n\
                                  AMP [default -12dBFS]\n\
